src/plugin.js

```diff
@@ -17,7 +17,16 @@ import mix from '../utils/mix.js';
  */
 export default class Plugin {
 	/**
-	 * Creates a new Plugin instance.
+	 * Creates a new Plugin instance. This is first step of a plugin initialization.
+	 * See also {@link #init} and {@link #afterInit}.
+	 *
+	 * A plugin is always instantiated after its {@link #requires dependencies} and the
+	 * {@link #init} and {@link #afterInit} methods are called in the same order.
+	 *
+	 * Usually, you'll want to put your plugin's initialization code in the {@link #init} method.
```

Comment:
That's one of the point of views. The other is that talking directly to the reader makes it simpler and more digestible. AFAIR we've been using this form from time to time.